File: Server_Systems_Development_Technology/Lab_Lichuha/Lab4/src/main/java/com/example/lab4/WebController.java
package com.example.lab4;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PathVariable;

@Controller
public class WebController {
    @GetMapping("/")
    public String MainPage() {
        return "MainMenu";
    }

    @GetMapping("doctors")
    public String DocIndex() {
        return "DoctorIndex";
    }

    @GetMapping("doctors/view{id}")
    public String DocGet(@PathVariable int id, Model model){
        model.addAttribute("docId", id);
        return "DoctorForm";
    }

    @GetMapping("patients")
    public String PatIndex() {
        return "PatientIndex";
    }

    @GetMapping("patients/view{id}")
    public String PatGet(@PathVariable int id, Model model){
        model.addAttribute("patId", id);
        return "PatientForm";
    }

    @GetMapping("appointments")
    public String AppIndex() {
        return "AppointmentsIndex";
    }

    @GetMapping("appointments/view")
    public String AppGet(){
        return "AppointmentForm";
    }
}

File: Server_Systems_Development_Technology/Lab_TessierAshpoule/3/lab3/src/main/kotlin/org/trsis/lab3/lab3/controller/WebController.kt
package org.trsis.lab3.lab3.controller

import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.PathVariable

@Controller
class WebController {
    @GetMapping("/")
    fun MainPage(): String {
        return "MainMenu"
    }

    @GetMapping("room/menu")
    fun DocIndex(): String {
        return "DoctorIndex"
    }

    @GetMapping("doctors/view{id}")
    fun DocGet(@PathVariable id: Int, model: Model): String {
        model.addAttribute("docId", id)
        return "DoctorForm"
    }

    @GetMapping("patients")
    fun PatIndex(): String {
        return "PatientIndex"
    }

    @GetMapping("patients/view{id}")
    fun PatGet(@PathVariable id: Int, model: Model): String {
        model.addAttribute("patId", id)
        return "PatientForm"
    }

    @GetMapping("appointments")
    fun AppIndex(): String {
        return "AppointmentsIndex"
    }

    @GetMapping("appointments/view")
    fun AppGet(): String {
        return "AppointmentForm"
    }
}
